addons/product/views/res_config_settings_views.xml

```diff
@@ -38,6 +38,30 @@
                         </div>
                     </div>
                 </xpath>
+                 <xpath expr="//div[@id='product_get_pic_setting']" position="replace">
```

Comment:
Couldn't this be part of `base_setup` directly?

Reply:
This way, the div is hardcoded in base_setup but doesn't appear until you install Product to avoid confusion of being able to configure something you don't have.